src/App.js

```diff
@@ -1,10 +1,41 @@
-import React from 'react';
+import React, { Component } from 'react';
+import MyButton, {UserInput} from './components/ToDoList.js';
 
-const App = () => (
-  <div>
-    <h2>Todo App</h2>
 
-  </div>
-);
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      title: 'To-Do',
+      task: '',
+      tasks: []
+    };
+  }
+
+  handleNewTask = () => {
+    let tasks = this.state.tasks;
+    tasks.push(this.state.task);
+    this.setState({"task": '', "tasks": tasks});
+  }
+
+  handleTaskString = event => {
+    this.setState({ [event.target.name]: event.target.value})
+  };
+
+  handleTaskDone = (event) => {
+    this.setState({ task: event.target.value});
+  };
+  
+  render() {
+    return (
+    <div>
+      <h1>{this.state.title}</h1>
+      <MyButton />
+      <UserInput />
```

Comment:
We need an onClick in here somewhere so that we can actually submit the task and add it to the list. Right now I can add to the input field but the button doesn't work yet.  Check out https://reactjs.org/docs/handling-events.html